### ISSUE TITLE
Traduções

### ISSUE THREAD

**Issue by prhost** [prhost/new-opencart#12]
A tradução será separada em três lugares:
- Dentro do core como de costume, mas sem sofrer alterações.
- Dentro da extensão, agora você pode adicionar suas traduções e também sobrepor traduções do core sem modifica-las diretamente.
- Dentro do tema. Acho essencial isso pois ao mesmo tempo que um front-end esta desenvolvendo o tema ele já pode desenvolvendo todo o translater sem mexer em controllers.

Uma coisa que eu gosto de usar, é uma função do Twig no theme que se chamada `lang()` nela você passa a palavra a ser buscada nos arquivos de tradução, sem precisar setar um monte de array na controller pra cada palavra a ser traduzida, exemplo de uso no html: 

``` html
<p>{{ lang('empty_name', 'catalog/product') }}</p>
```

![screen shot 2016-01-27 at 2 49 46 pm](https://cloud.githubusercontent.com/assets/3984760/12620898/418734e2-c505-11e5-9fc8-b5d77edeee8b.png)

**Comment by ghost** [comment]
Muito boa a ideia.
Se usado o Twig no tema, ficaria realmente bom.
Achei interessante o recurso de mudar a tradução do core sem sobrepor, algo similar ao vQmod, mas creio que sem precisar usar ele.

**Comment by prhost** [comment]
Exato, se por exemplo em sua extensão tiver translater que conflite com os nativos, o loader vai dar prioridade as suas, assim você por exemplo podemos criar uma extensão só contendo traduções e mante-lo em um repositório público para a comunidade.

**Comment by felipoantonoff** [comment]
Assim fica ótimo, já tive muito trabalho criando xml para mudar a linguagem nativa, como muitos mexem, tinha que adaptar. O ideal é ver a identificação pelo nome usado na tradução, mesmo editada da para trocar por outra.

**Comment by prhost** [comment]
@FelipoCodeMarket desculpe, não entendi kkk, sou meio lento, explique por gentileza a partir do "O ideal é ver a identificação..."

**Comment by victorhramos** [comment]
E sobre utilizar i18n?
